pkg/metrics/metrics.go

```diff
@@ -476,7 +474,7 @@ var (
 )
 
 func init() {
-	MustRegister(prometheus.NewProcessCollector(os.Getpid(), Namespace))
+	MustRegister(prometheus.NewProcessCollector(prometheus.ProcessCollectorOpts{Namespace: "cilium"}))
```

Comment:
This previously used the variable name, it should probably do so here too:

```suggestion
	MustRegister(prometheus.NewProcessCollector(prometheus.ProcessCollectorOpts{Namespace: Namespace}))
```